unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Font/Editor/FontAssetBundleGenerator.cs

```diff
@@ -0,0 +1,54 @@
+using DCL.ABConverter;
+using System;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+public class FontAssetBundleGenerator : MonoBehaviour
```

Comment:
Do we still need it? Can we just delete it?

Reply:
Done!